src/devices/src/virtio/net/device.rs

```diff
@@ -256,9 +259,15 @@ impl Net {
     }
 
     fn signal_rx_used_queue(&mut self) -> result::Result<(), DeviceError> {
-        if self.rx_deferred_irqs {
+        if !self.rx_deferred_irqs {
```

Comment:
Can we delete `rx_deferred_irqs` and rely only on `prepare_kick()` ? It's a tradeoff for code simplicity. It will degrade the performance when the guest doesn't use notification suppression but usually this shouldn't happen. I think we did something similar for the block device.

Reply:
I got rid of the `rx_deferred_irqs` altogether. I 'm now relying only on `prepare_kick()` to trigger interrupts